src/components/CommonComponents/Hero/index.module.scss

```diff
@@ -49,7 +49,7 @@
       flex-direction: column;
       position: relative;
 
-      & > p {
+      &>p {
```

Comment:
I think this is making the build fail.

Reply:
<img width="712" alt="image" src="https://user-images.githubusercontent.com/30056453/196680598-6ae07abf-97b3-4972-86a5-7627ab150b22.png">
I ran the tests again, I couldn't see a problem, but I fixed the point you mentioned.

Reply:
I meant the CI build, not Gatsby build xD